news/2019-10-20-beatmap-pack-management-update-1.md

```diff
@@ -0,0 +1,63 @@
+
+### Best of 2010–2011 pack
+
+*created by [Dada](https://osu.ppy.sh/users/9119507), [\_p0ke\_](https://osu.ppy.sh/users/5434711), [Halfslashed](https://osu.ppy.sh/users/4598899), [Uberzolik](https://osu.ppy.sh/users/1314547), and [Cynplytholowazy](https://osu.ppy.sh/users/3901754)*  
+*written by [\_p0ke\_](https://osu.ppy.sh/users/5434711)*  
+[Download](https://osu.ppy.sh/beatmaps/packs/1733)  
+
+Ever wanted to see what popular maps looked like in earlier eras of the game? Well now you can! Forged together by members of the Beatmap Pack Management team, you can check out some of the best maps that the game had to offer during 2010 and 2011 in one single pack. While there were a lot of great maps the team didn't have enough room to include, hopefully you can find something on this list that you will enjoy. Play through them all and experience a blast from the past!
+
+### osu!dan project
+
+*written by [celerih](https://osu.ppy.sh/users/4696296)*  
+
+The osu!dan project is a brand new community created project that seeks to provide a sort of "campaign mode" to the game. Players will progress through levels of increasing difficulty in a skill of their choice, such as aim, stamina, accuracy, et cetera.
+
+Each of the dans will contain 20 levels, each of increasing difficulty. In order to complete a level, the player must reach the requirements on the maps presented, like reaching a certain accuracy threshold or attaining a full combo. For now, the release of dan levels is purely for fun, a way to practice, or testing yourself, but a more official implementation is planned for the future. wink wink
+
+The first of the dans is now available to download on the beatmap packs page. The skill tested in this first release is aim and is sure to push your circle clicking limits. The requirements for this dan are quite simple; just full combo 10 of the 11 maps in a level to progress to the next. Once again, this for now is just for your own practice, or to compare yourself with your friends—there is no implementation for keeping track of levels as of now.
+
+If you wish to join the osu!dan server to assist with playtesting future packs and stay up to date, [you can join the server!](https://discord.gg/b7cVmGd)
+
+### October Loved packs
+
+*written by [Stefan](https://osu.ppy.sh/users/626907)*
+
+As it is time again to bundle a few beatmaps, there are some new packs for loved beatmaps available since yesterday. I want to take the chance to explain why I did not bundle the latest maps from Loved already: Since there is always a small chance that a beatmap can become unloved (and therefore unranked) it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.
```

Comment:
```suggestion
As it is time to bundle a few beatmaps again, there are some new packs for Loved beatmaps available since yesterday. I want to take the chance to explain why I did not bundle the latest maps from Loved already - since there is always a small chance that a beatmap can get unloved (and therefore unranked) it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.
```